code/datums/achievements/boss_achievements.dm

```diff
@@ -66,12 +66,6 @@
 	desc = "You've now ruined years of mythical storytelling."
 	database_id = BOSS_MEDAL_WENDIGO
 
-/datum/award/achievement/boss/king_goat_kill
-	name = "King Goat Killer"
-	desc = "The king is dead, long live the king!"
-	database_id = BOSS_MEDAL_KINGGOAT
-	icon = "goatboss"
```

Comment:
there is no support of removing achievements iirc so bring these back

Reply:
fixed